services/app-web/src/components/SubmissionSummarySection/RateDetailsSummarySection/RateDetailsSummarySection.tsx

```diff
@@ -39,11 +39,29 @@ export const RateDetailsSummarySection = ({
                                     ? 'Rating period of original rate certification'
                                     : 'Rating period'
                             }
-                            data={`${dayjs(submission.rateDateStart).format(
-                                'MM/DD/YYYY'
-                            )} to ${dayjs(submission.rateDateEnd).format(
-                                'MM/DD/YYYY'
-                            )}`}
+                            data={
+                                <>
+                                    <time
+                                        dateTime={dayjs(
+                                            submission.rateDateStart
+                                        ).format('YYYY-MM-DD')}
+                                    >
+                                        {dayjs(submission.rateDateStart).format(
+                                            'MM/DD/YYYY'
+                                        )}
+                                    </time>{' '}
+                                    to{' '}
```

Comment:
i would put the to in a `<span>{' to '}</span>`

to me random spaces `'  '` outside an element are a problem in jsx - they are so easy to accidentally delete (or add more spaces accidental with prettier) and then change the way the UI looks. you can also use `&nbsp;` for spaces fyi